serverless/aws/sam/src/main/scala/sttp/tapir/serverless/aws/sam/Printer.scala

```diff
@@ -0,0 +1,167 @@
+package sttp.tapir.serverless.aws.sam
+
+import Printer._
+import io.circe._
+import java.io.StringWriter
+import org.yaml.snakeyaml.DumperOptions
+import org.yaml.snakeyaml.emitter.Emitter
+import org.yaml.snakeyaml.nodes._
+import org.yaml.snakeyaml.resolver.Resolver
+import org.yaml.snakeyaml.serializer.Serializer
+import scala.collection.JavaConverters._
+
+// modified stringNode to handle !Ref tags
+final case class Printer(
```

Comment:
another follow-up task: add a PR to https://github.com/circe/circe-yaml which would propagate this change.

Didn't we have another modification elsewhere with a "fixed" `isBad` method?

Reply:
We thought about fixing `isBad` to handle string literals with some special characters automatically but we finally settled for configuring that in open api `toYaml` methods